dlp/api/Snippets/InspectStorageWithSampling.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START dlp_inspect_gcs_with_sampling]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Google.Cloud.PubSub.V1;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class InspectStorageWithSampling
+{
+    public static async Task<DlpJob> InspectAsync(
+        string projectId,
+        string gcsFileUri,
+        string topicId,
+        string subId,
+        Likelihood minLikelihood = Likelihood.Possible,
+        IEnumerable<InfoType> infoTypes = null)
+    {
+
+        // Instantiate the dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct Storage config.
```

Comment:
Please refer to the Java example provided. I feel the comments there are much more helpful.
The comments here are stating what the code is already doing and adds little value.
I suggest we should emulate that for comments. Here and in other samples as well.

Reply:
Done Thanks.